src/cpp/session/resources/dependencies/r-packages.json

```diff
@@ -605,7 +547,7 @@
                 "jsonlite",
                 "knitr",
                 "learnr",
-                "markdown",
```

Comment:
`learnr` says it depends on `rmarkdown` https://github.com/rstudio/learnr/blob/v0.10.1/DESCRIPTION was this a typo ?

Reply:
I don't think so, `learnr` tutorials are basically just special R Markdown documents, and so rendering and running those does require the `rmarkdown` package.